ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m

```diff
@@ -38,16 +38,13 @@ @implementation WebRTCModule (RTCMediaStream)
  */
 typedef void (^NavigatorUserMediaSuccessCallback)(RTCMediaStream *mediaStream);
 
-- (RTCMediaConstraints *)defaultMediaStreamConstraints {
-  NSDictionary *mandatoryConstraints
-      = @{ kRTCMediaConstraintsMinWidth     : @"1280",
-           kRTCMediaConstraintsMinHeight    : @"720",
-           kRTCMediaConstraintsMinFrameRate : @"30" };
-  RTCMediaConstraints* constraints =
-  [[RTCMediaConstraints alloc]
-   initWithMandatoryConstraints:mandatoryConstraints
-   optionalConstraints:nil];
-  return constraints;
+
+- (NSMutableDictionary *)defaultMediaStreamConstraints {
+    NSMutableDictionary *mandatory = [[NSMutableDictionary alloc]init];
+    [mandatory setObject:@"720" forKey:@"maxHeight"];
+    [mandatory setObject:@"1280" forKey:@"maxWidth"];
+    [mandatory setObject:@"30" forKey:@"maxFrameRate"];
```

Comment:
instead of repeating these string literals over and over I'd like to see a static constant.